### DISCUSSION TITLE
Release: v11.0.0

### DISCUSSION THREAD

**Discussion by georglauterbach** [docker-mailserver/docker-mailserver#2509] [Announcements]
This is the thread for the upcoming version of DMS: `v11.0.0`.

**Comment by jayknyn** [comment]
Hi Georg and team, I'm excited for the v11 release. 

I've had a look at the [v11 milestone ](https://github.com/docker-mailserver/docker-mailserver/milestone/10)and the [release PR](https://github.com/docker-mailserver/docker-mailserver/pull/2537). Is the draft chanelog for V11 being documented somewhere or is the milestone and PR it for now? I just want to prepare for the upgrade and see if there are any breaking changes. I will pull down and run the edge image in a test environment when it's released. 

Thanks for all that you do!

**Reply by georglauterbach** [reply]
The release PR you linked will contain a full changelog when all other PRs for v11 are merged. You will then be able to get a nice overview.

There will also be an additional feature freeze for one week until v11 is merged so we can thoroughly test the `:edge` tag.

If I remember, I will keep this thread up to date :)

**Reply by georglauterbach** [reply]
There is now a more or less complete changelog available in #2537. Feature-freeze is in-place at least until 27th April 2022.

**Reply by jayknyn** [reply]
Thank you, Georg!

**Comment by casperklein** [comment]
> Is the draft chanelog for V11 being documented somewhere

No. But you can take a look at the milestones or merged PRs since the last release.

> I will pull down and run the edge image in a test environment when it's released.

The edge image is a "rolling release". It's updated every time, a PR is merged. So you can test it at any time you want.
With the exception of the four outstanding PRs and possible bugfixes that might be necessary after testing, the edge image represents the upcoming v11 version.

**Comment by georglauterbach** [comment]
`v11.0.0` was just released: <https://github.com/docker-mailserver/docker-mailserver/releases/tag/v11.0.0>

Also have a look at the [`CHANGELOG`](https://github.com/docker-mailserver/docker-mailserver/blob/v11.0.0/CHANGELOG.md).
